### ISSUE TITLE
Remove embedded normalize

### ISSUE THREAD

**Issue by nwmcsween** [BafS/Gutenberg#23]
Instead of embedding normalize gutenberg should probably say it's required in the documentation then users could use a newer version of normalize

**Comment by BafS** [comment]
Yes it could do it as a peer dependencies, feel free to submit a patch.

**Comment by fanchyfanch** [comment]
It appears the link in the gutenberg.scss is wrong. Instead of:

> @import '../node_modules/normalize.css/normalize';

it should be:

> @import '../../normalize.css/normalize';

**Comment by BafS** [comment]
@fanchyfanch the path is correct, you can try to compile it with `npx gulp`. Do you include this file from another project ?

What we could do is use the tilda `~` (`@import '~normalize.css/normalize'`) like webpack but it's not natively handled with gulp-sass.

**Comment by fanchyfanch** [comment]
Actually, I'm loading gutenberg as a dependency in my project from SASS:

> @import '~gutenberg-css/scss/gutenberg';

And when I compile my project with Webpack Encore, it throws an error "unable to find normalize.css". 

I had to make an intermediary file to work as the "gutenberg.scss" entrypoint and to load every sub-files, including normalize.css like:

> @import '~normalize.css/normalize';

**Comment by BafS** [comment]
@fanchyfanch A new version was published (v0.6.0)
